### PR TITLE
Pass through retries / retry_delay to remote_file

### PR DESCRIPTION
**Pull request by aegixx**
### Description

Pass through retries / retry_delay to remote_file resource.

Obvious fix

### Issues Resolved

https://github.com/chef-cookbooks/aws/issues/237

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @aegixx

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

File: resources/s3_file.rb [hunk 1]
```diff
@@ -34,6 +34,8 @@
 attribute :force_unlink, kind_of: [TrueClass, FalseClass], default: false
 attribute :manage_symlink_source, kind_of: [TrueClass, FalseClass]
 attribute :sensitive, kind_of: [TrueClass, FalseClass], default: false
+attribute :retries, kind_of: [Integer, FalseClass], default: 0
```
Comment (tas50) [inline resources/s3_file.rb]: Do these actually need the falseclass?

Comment (aegixx) [inline resources/s3_file.rb]: Good catch, too quick to copy/paste!

**Comment by tas50** [comment]
@aegixx I merged in the retries logic in #246.  I can merge in the delay logic if you want to rebase this.

**Comment by tas50** [comment]
Closing until we can get a rebase on this. I'll gladly merge it with the rebase

**Comment by tas50** [comment]
I take it back. I just did the rebase and merge on this.

**Comment by aegixx** [comment]
Ha, thank you - I've had this on my to-do list for weeks!  🍻

**Pull request merged**